install_gdb.sh

```diff
@@ -3,13 +3,19 @@ set -e
 
 LIBDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
+CROSS_TC_PATH=${SYSROOT:=${HOME}/x-tools}
 CROSS_TC=${CROSS_TC:=arm-kobo-linux-gnueabihf}
 
-SYSROOT=${SYSROOT:=/home/${USER}/kobo/x-tools/${CROSS_TC}/${CROSS_TC}/sysroot}
```

Comment:
That's the main problem, for some reasons the SYSROOT=${SYSROOT:=....} does NOT work anymore.
Maybe because SYSROOT's value is empty string ?
Regardless testing with if [ ! -z "${SYSROOT}" ]; instead fixed the issue for me. 
Without it, it will try to install every libs in ~/x-tools/usr
Resulting in qt failing to compile due to numerous linking issue (because sysroot and pkconfig is broken)